src/force/CabanaPD_Force_PMB.hpp

```diff
@@ -397,7 +507,7 @@ class Force<MemorySpace, ForceModel<LinearPMB, Elastic, ModelParams...>>
     template <class PosType, class WType, class ParticleType,
               class ParallelType>
     double computeEnergyFull( WType& W, const PosType& x, const PosType& u,
```

Comment:
Missing linear PMB

Reply:
@kinanbezem7 Last changes needed are linear PMB + LPS stresses that match PMB

Reply:
I'm trying to run the crack_branching problem just to check that stress is working. but i don't see it in the SILO output files. Does it not fall under OtherFields here in particles.hpp? 

```
#ifdef Cabana_ENABLE_SILO
        Cabana::Grid::Experimental::SiloParticleOutput::
            writePartialRangeTimeStep(
                "particles", local_grid->globalGrid(), output_step, output_time,
                0, n_local, getPosition( use_reference ), sliceForce(),
                sliceDisplacement(), sliceVelocity(),
                std::forward<OtherFields>( other )... );
```

Reply:
You've reminded me of a Silo-specific issue with matrix fields. As I recall they just don't show up in paraview. If you open an issue on your hdf5 problems I can try to help there.